lib/ui/painting/image_decoder_unittests.cc

```diff
@@ -34,7 +34,8 @@ class TestIOManager final : public IOManager {
                 : nullptr),
         unref_queue_(fml::MakeRefCounted<SkiaUnrefQueue>(
             task_runner,
-            fml::TimeDelta::FromNanoseconds(0))),
+            fml::TimeDelta::FromNanoseconds(0),
+            gl_context_)),
```

Comment:
It looks like there are several places other than this in the code where a context is not passed to the `SkiaUnrefQueue` constructor. If failing to pass the context where needed can result in crashes, then that usage of optional arguments is probably not in agreement with the Google style guide. I'd like to hear thoughts from @jason-simmons on this.

Reply:
+1 to making this a require parameter.

Reply:
Making it a required parameter would be ideal.

AFAICT making it optional is a workaround for scenarios like unit tests where it may be impractical to provide something resembling a GrContext.  It would be preferable to avoid that if possible.

Reply:
The class was written as a template class so that tests could explicitly create their own idea of what a context is.

We should make it non-optional and consider adding a DCHECK that it's not nullptr in the constructor.